Mathlib/Combinatorics/SimpleGraph/Operations.lean

```diff
@@ -29,17 +30,23 @@ section ReplaceVertex
 
 /-- The graph formed by forgetting `t`'s neighbours and instead giving it those of `s`. The `s-t`
 edge is removed if present. -/
-abbrev replaceVertex : SimpleGraph V where
+def replaceVertex : SimpleGraph V where
   Adj v w := if v = t then if w = t then False else G.Adj s w
                       else if w = t then G.Adj v s else G.Adj v w
   symm v w := by dsimp only; split_ifs <;> simp [adj_comm]
 
+@[simp]
```

Comment:
Do you **always** want to simplify LHS to RHS?

Reply:
Fixed; I'm now seeing the benefits of keeping definitions folded through `simp` calls. I managed to rewrite `CliqueFree.replaceVertex` to run even faster than before.